app.js

```diff
@@ -9,19 +9,31 @@ const connection = require("./db/connection");
 //require controllers
 const getTopics = require("./controllers/topics.controller")
 const getEndpoints = require("./controllers/endpoints.controller")
+const getArticleById = require("./controllers/articles.controller")
 
-//ENDPOINTS
+////ENDPOINTS////
 app.get("/api/topics", getTopics);
 app.get("/api", getEndpoints);
+app.get("/api/articles/:article_id", getArticleById);
 
-//ERROR HANDLING
-//Bad path
+////ERROR HANDLING////
+//400 Bad Path
+//No err parameter as should be triggered where no endpoint reached
 app.use((req, res, next) => {
-    res.status(400).send({ msg: "Path does not exist."});
+    console.log("BAD PATH BABY")
```

Comment:
Maybe don't have this console log in production-level code, i.e. code you believe to be final and ready for review by a senior developer ;)